test/README.md

```diff
@@ -1,13 +1,13 @@
 # Tests
 
-Run the test docker compose:
+To run these tests in docker compose:
 
 ```
-docker-compose -f docker-compose.yml -f test.docker-compose.yml up
+docker compose run test
 ```
 
-And then run the tests:
+To clean up these containers afterwards, the `--profile=test` argument needs to be passed to docker:
 
 ```
-docker exec -it blackbird-api yarn test
+docker compose --profile=test down
```

Comment:
there's no way to do this automatically?

Reply:
There is `docker compose --profile=test up --exit-code-from=test`, although that stops *every* container after exiting, not just the containers in `--profile=test`. Meanwhile, `docker compose run --rm test` is another option, but it only removes the `test` container and still leaves redis/postgres/minio up - so I haven't found a perfect solution.